class/defaults.yml

```
@@ -33,3 +33,32 @@ parameters:
     generatedDefaultRoleBindingInNewNamespaces:
       bindingName: admin
       clusterRoleName: admin
+
+    generatedResourceQuota:
+      name: organization
+      synchronize: false
+      hard:
+        requests.cpu: "1"
+        requests.memory: 4Gi
+        limits.cpu: "2"
+        limits.memory: 8Gi
```

We have some values defined in our requirements: https://kb.vshn.ch/appuio-cloud/references/quality-requirements/performance/resource-quota.html
Whether these values are sensible is another question, but please keep them in sync at least.
```suggestion
        # See https://kb.vshn.ch/appuio-cloud/references/quality-requirements/performance/resource-quota.html
        requests.cpu: "1"
        requests.memory: 4Gi
        limits.cpu: "2"
        limits.memory: 8Gi
```

---

As "choose reasonable defaults" is explicitly out of scope, I just chose something arbitrary.  But sure I can switch to these for now.